Configuration/Test/test.py

```diff
@@ -0,0 +1,534 @@
+        threshold = cms.untracked.string('WARNING')
+    ),
+    infos = cms.untracked.PSet(
+        threshold = cms.untracked.string('INFO')
+    ),
+    debugs = cms.untracked.PSet(
+        threshold = cms.untracked.string('DEBUG'),
+        INFO = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        DEBUG = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        TotemRP = cms.untracked.PSet(
+            limit = cms.untracked.int32(1000000)
+        ),
+        ForwardSim = cms.untracked.PSet(
+            limit = cms.untracked.int32(1000000)
+        )
+    )
+)'''
+
+# Specify the maximum events to simulate
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(1000)
+)
+
+# Configure the output module (save the result in a file)
+process.o1 = cms.OutputModule("PoolOutputModule",
+    outputCommands = cms.untracked.vstring('keep *'),
+    fileName = cms.untracked.string('file:test_1k.root')
+)
+process.outpath = cms.EndPath(process.o1)
+
+################## STEP 1 - process.generator
+process.source = cms.Source("EmptySource")
+
+# Use random number generator service
+#process.load("Configuration.TotemCommon.RandomNumbers_cfi")
+
+# particle generator paramteres
+process.load("Configuration.StandardSequences.Services_cff")
+
+process.load("Configuration.Generator.FlatLogKsiLogT_pp_13TeV_cfi")
+process.RandomNumberGeneratorService = cms.Service("RandomNumberGeneratorService",
+    g4SimHits = cms.PSet(initialSeed = cms.untracked.uint32(9876)),
+    SimG4Object = cms.PSet(initialSeed =cms.untracked.uint32(9876)),
+    RPSiDetDigitizer = cms.PSet(initialSeed =cms.untracked.uint32(137137)),
+    sourceSeed = cms.PSet(initialSeed =cms.untracked.uint32(98765)),
+    generator = cms.PSet(initialSeed = cms.untracked.uint32(98766)),
+    SmearingGenerator = cms.PSet(initialSeed =cms.untracked.uint32(3849)),
+    T2Digis = cms.PSet(initialSeed =cms.untracked.uint32(98765)),
+    T2MCl = cms.PSet(initialSeed =cms.untracked.uint32(24141)),
+    RPFastStationSimulation = cms.PSet(initialSeed =cms.untracked.uint32(12)),
+    RPFastFullSimulation = cms.PSet(initialSeed =cms.untracked.uint32(13)),
+    mix = cms.PSet(initialSeed = cms.untracked.uint32(24141)),
+    LHCTransport = cms.PSet(initialSeed = cms.untracked.uint32(24143), engineName = cms.untracked.string('TRandom3')
+  )
+
+)
+################# STEP 2 process.SmearingGenerator
+process.SmearingGenerator = cms.EDProducer("GaussEvtVtxEnergyGenerator",
+    src   = cms.InputTag("generator"),
+    MeanX = cms.double(0.0),
+    MeanY = cms.double(0.0),
+    MeanZ = cms.double(0.0),
+    SigmaX = cms.double(0.000001),
+    SigmaY = cms.double(0.000001),
+    SigmaZ = cms.double(0.000001),
+    TimeOffset = cms.double(0.0)
+)
+# declare optics parameters
+#process.load("Configuration.TotemOpticsConfiguration.OpticsConfig_6500GeV_0p8_145urad_cfi")
+process.BeamOpticsParamsESSource = cms.ESSource("BeamOpticsParamsESSource",
```

Comment:
Do we have ANY file with preconfigured optics in the repo ? File like `OpticsConfig_6500GeV_0p8_145urad_cfi`

Reply:
There is no such file, but found similar configuration file at Eliza's REPO:
https://github.com/elizamelo/CMSTOTEMSim/blob/CMSTOTEM-v01/CMSSW_7_0_4/src/Configuration/TotemOpticsConfiguration/python/OpticsConfig_6500GeV_0p8_145urad_cfi.py
Can create similar one.

Reply:
better to keep it like that now.